app/containers/ActionView/index.js

```diff
@@ -100,23 +212,24 @@ const mapStateToProps = (state, props) => ({
           path: 'measure_categories',
           key: 'category_id',
           where: {
-            action_id: props.params.id,
+            measure_id: props.params.id,
           },
         },
       },
       out: 'js',
     },
   ),
-  // all recommendations, listing connection if any
+  // all connected recommendations
   recommendations: getEntities(
     state, {
       path: 'recommendations',
       out: 'js',
       join: {
```

Comment:
join not extend as we only need those that are associated and not all optional associations